### PR TITLE
PLAT-128528: Merge updated kitten browser related source to develop

### PR DESCRIPTION
**Pull request by YunhoLee0**
Enact-DCO-1.0-Signed-off-by: Yunho Lee (yunho2.lee@lge.com)

Update dependencies for enact, sandstone (to public), react (to 17)

### CODE REVIEW

File: enact-all-samples/src/App/App.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import kind from '@enact/core/kind';
 import MoonstoneDecorator from '@enact/moonstone/MoonstoneDecorator';
 import PropTypes from 'prop-types';
-import React from 'react';
+import React from 'react'; // eslint-disable-line no-unused-vars
```
Comment (MikyungKim) [inline enact-all-samples/src/App/App.js:4]: Let's modify travis.yml file instead of disabling eslint.

Comment (YunhoLee0) [reply enact-all-samples/src/App/App.js]: I've temporarily changed the travis.yml file and lint script to work for only two folders (tutorial-hello-enact, tutorial-kitten-browser).

**Review by MikyungKim — APPROVED** [review]
LGTM

File: enact-all-samples/src/App/App.js (hunk 1 above)
Comment (MikyungKim) [reply enact-all-samples/src/App/App.js]: 👍

**Pull request merged**